### DISCUSSION TITLE
Is there a way to pull compound data with laps data?

### DISCUSSION THREAD

**Discussion by hawai50** [br-g/openf1#6] [General]
Hi guys, firstly thank you so much for this amazing service you offer! I'm just getting to grips at the moment. One thing I'd like to record is the tyre compound relative to the lap time. I realise tyre compound seems to sit under 'stint' and lap time under 'lap'. Would there be any sensible way to combine those data sets, or record compound as part of the lap time? I'm guessing it's a no as the 'stint' section records laps from-to rather than the compound on specific lap. Many thanks in advance for suggestions.

**Comment by br-g** [comment]
Hello! Thank you for your kind words.
Unfortunately, there isn't a straightforward method to directly link tyre compound information to each lap. You'll need to manually combine these datasets for now.

**Comment by hawai50** [comment]
Thanks so much for replying, I've managed to do it with python and combine
the datasets by searching the tyre start end range.
